### PR TITLE
[ENH] This adds the brain and the electrodes in fsaverage

### PR DESCRIPTION
**Pull request by alexrockhill**
The brain is for the SDR and the electrodes in fsaverage are for if it's too slow to compute in the example.

### CODE REVIEW

**Comment by alexrockhill** [comment]
Wait, one contact was double marked on the same place, fixing it now

**Comment by alexrockhill** [comment]
Ok fixed, good to go

File: seeg/sample_seeg_electrodes_fsaverage.tsv [hunk 1]
```diff
@@ -0,0 +1,128 @@
+	name	R	A	S
```
Comment (larsoner) [inline seeg/sample_seeg_electrodes_fsaverage.tsv:1]: This is no longer used in the example, can we get rid of it?

**Comment by alexrockhill** [comment]
Ok, I added back the final result because if we are putting code in `code-blocks` not to be executed, we'll need this. And it's only a few KB anyway.

I think this is ready to merge, everything works.

**Comment by alexrockhill** [comment]
Good thing it wasn't merged, LAMY 2 was a bit off. Just fixed it, good to go. Results are good and should execute in under a minute.

File: seeg/sample_seeg_electrodes_fsaverage.tsv [hunk 2]
```diff
@@ -0,0 +1,128 @@
...
+0	LACN 1	-7.0	14.5	25.0
```
Comment (larsoner) [inline seeg/sample_seeg_electrodes_fsaverage.tsv:2]: I still dream of a day we don't need this, but it's tiny so I'll merge so we can actually see what CircleCI thinks!

**Pull request merged**